src/js/simplebox.js

```diff
@@ -58,7 +59,10 @@
 
                 var $this = $(this);
                 var imageSRC = $this.attr("src");
-                if (~$this.attr("class").indexOf("hq")) imageSRC = imageSRC.substring(0, imageSRC.lastIndexOf(".")) + settings.postfix + imageSRC.substring(imageSRC.lastIndexOf("."));
+                if (~$this.attr("class").indexOf(settings.hqClass)) {
+                    let dotIndex = imageSRC.lastIndexOf(".");
```

Comment:
Hey, could you switch the `let` to a `var` for now? The project isn't in ES6 right now using `let` just messes with `gulp`.

Reply:
1. Yes, I do. Basically, if someone wanted to use this new feature and had a lot of images with simplebox, they could enable it for 
only _some_ of the images, and this way other ones (that don't have a high-res version yet) won't break.
2. Yes, I'll do it a bit later.
3. Ok, I'll change it to `var`.